docs/whatsnew/_index.md

```diff
@@ -4,6 +4,10 @@ shorttitle: What's new
 weight: 15
 ---
 
+## Release 0.5 (2021-04-19) {#oe05}
+
+One Eye allows you to attach multiple peer clusters to a central observer cluster to make your metrics available centrally, and store in object storage in a cost and storage-efficient way. One Eye automates several steps of connecting and maintaining the peer clusters, making it possible to operate your monitoring infrastructure with a low overhead. For details on how One Eye collects metrics from multiple clusters, see {{% xref "/docs/one-eye/multicluster/_index.md" %}}.
+
```

Comment:
we should also add external ingress + authentication with pomerium + role based authorization as a note here, although we don't have the docs for it just yet